src/components/Compare/Selections.tsx

```diff
@@ -0,0 +1,76 @@
+import { Grid, Button } from "@trussworks/react-uswds";
+import { ReactComponent as Close } from "../../images/close.svg";
+import { ReactComponent as CaretDown } from "../../images/caret-down.svg";
+import { useState, Dispatch, SetStateAction } from "react";
+import { CareProvider } from "../../types";
+import { CompareProviders } from "../../pages/Compare";
+
+type SelectionsProps = {
+  providers: CompareProviders;
+  setProviders: Dispatch<SetStateAction<CompareProviders>>;
+};
+
+function Selections({ providers, setProviders }: SelectionsProps) {
+  const [showSelections, setShowSelections] = useState(true);
+  return (
+    <Grid row>
+      {showSelections && (
+        <>
+          <Selection
+            provider={providers.providerA}
+            clearFunc={() => setProviders({ providerB: providers.providerB })}
+          />
+          <Selection
+            provider={providers.providerB}
+            clearFunc={() => setProviders({ providerA: providers.providerA })}
+          />
+        </>
+      )}
+
+      <button
+        className="tablet:display-none margin-0 bg-white text-black padding-x-1 padding-y-2 radius-lg width-full"
+        onClick={() => setShowSelections(!showSelections)}
+      >
+        # Location selected
```

Comment:
having the selected providers prop as a list will also make the count here a lil more straightforward